src/main/java/net/fuzzycraft/botanichorizons/patches/CraftingPatches.java

```diff
@@ -1746,6 +1747,36 @@ public static void applyPatches() {
         ModCraftingRecipes.recipeShimmerwoodPlanks = BotaniaAPI.getLatestAddedRecipe();
         GregtechPatches.addStairs(ModFluffBlocks.shimmerwoodPlankStairs, new ItemStack(ModBlocks.shimmerwoodPlanks));
 
+        // Quartz Recipes
+		if(ConfigHandler.darkQuartzEnabled)
+            ModCraftingRecipes.recipeDarkQuartz = addQuartzRecipes(0, Items.coal, ModFluffBlocks.darkQuartz, ModFluffBlocks.darkQuartzStairs, ModFluffBlocks.darkQuartzSlab);
+        addQuartzRecipes(1, null, ModFluffBlocks.manaQuartz, ModFluffBlocks.manaQuartzStairs, ModFluffBlocks.manaQuartzSlab);
+        ModCraftingRecipes.recipeBlazeQuartz = addQuartzRecipes(2, Items.blaze_powder, ModFluffBlocks.blazeQuartz, ModFluffBlocks.blazeQuartzStairs, ModFluffBlocks.blazeQuartzSlab);
+
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.quartz, 8, 3),
+                        "QQQ", "QCQ", "QQQ",
+                        'Q', "manaQuartz",
+                        'C', new ItemStack(Blocks.red_flower, 1, 2)));
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.quartz, 8, 3),
+                        "QQQ", "QCQ", "QQQ",
+                        'Q', "manaQuartz",
+                        'C', new ItemStack(Blocks.red_flower, 1, 7)));
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.quartz, 8, 3),
+                        "QQQ", "QCQ", "QQQ",
+                        'Q', "manaQuartz",
+                        'C', new ItemStack(Blocks.double_plant, 1, 1)));
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.quartz, 8, 3),
+                        "QQQ", "QCQ", "QQQ",
+                        'Q', "manaQuartz",
+                        'C', new ItemStack(Blocks.double_plant, 1, 5)));
```

Comment:
The quartz part was taken directly from the original botania code but could be changed directly in the botania files instead of here, I place them here just in case in the future an assembler recipe is added to have everything more together ( also because I dont know how to make what you are asxking for)

Reply:
- Because someone else did it does not make it a good idea :wink:
- Recipe changes do not go in the base Botania code, that would affect every user looking for a bugfix version.
- Constants you can use are available here: https://github.com/GTNewHorizons/Botanic-horizons/blob/master/src/main/java/net/fuzzycraft/botanichorizons/util/Constants.java
- Flowers are defined and colour-tagged here: https://github.com/GTNewHorizons/Botanic-horizons/blob/master/src/main/java/net/fuzzycraft/botanichorizons/patches/OredictPatches.java

Reply:
@combusterf  Hey I´m going to work now in this, two things, the sunnyquartz should stay with the sunflower because the name, and the lavender quartz uses lavender from BoP, should I change it to use vanilla lilac since those are renovable?

Reply:
Also there are two variants of Smokey Quartz, one from Botania and the other from Thaumic Tinkering, with the same block variants but diferent crafts and uses, if could be nice if all the crafts where oredic using by basis for the quartz the thaumic tinkers one, but I dont know how to make that ( so I have tell it to Alastor)

Reply:
Sunny quartz is renewable as is, though I did consider making a expensive-but-simple version using sunnarium pieces. Lavender quartz should definitely keep the lavender version, but if you really want to give it a use and need renewability, consider the botany lavender or the pink tinted comb from lavender bees to give people a bit of a challenge to get there. It's not like the BoP lavender plant is particularly rare one.

Smokey quartz is currently set up to cause no conflicts or bypasses at the cost of keeping the two kinds completely separate, but some coordination and unification could be welcome.

None of this is necessary for passing the code review though ;)

Reply:
@combusterf Could you check the ping at magic-dev when you have time? Also I have no use for the lavender quartz in mind, for all the rest yes, so is okay with that, PS I'm sorry for having this pr open for so long, but between the little free time I have and the fact that there is one thing that I don't quite understand (the discord ping thing) I'm a little slow